Add an option to upgrade an existing app (fix #1131) (#1138)
nativefier/nativefier: docs/api.md

Prints the version of your `nativefier` install.

#### [upgrade]

```
--upgrade <value>
```

> The path to an application packaged with a previous version of Nativefier, to be upgraded to the latest version of Nativefier. This may be the `.exe` file in Windows, the generated executable in Linux, or the generated `.app` for Mac. This will need to be contained within the full context that it was generated in previously (i.e., on Windows and Linux, the exe file should still be in the folder with the generated resources directory). This option will attempt to extract all existing options from the old app.

Reviewer: ```suggestion
This option will attempt to extract all existing options from the old app, and upgrade it using the current Nativefier CLI.

The provided path must be the "executable" of an application packaged with a previous version of Nativefier, and to be upgraded to the latest version of Nativefier. "Executable" means: the `.exe` file on Windows, the executable on Linux, or the `.app` on macOS. The executable must be living in the original context where it was generated (i.e., on Windows and Linux, the exe file must still be in the folder containing the generated `resources` directory).
```

--upgrade <pathToExistingApp>
```

*NEW IN 43.1.0*

This option will attempt to extract all existing options from the old app, and upgrade it using the current Nativefier CLI.

**IMPORTANT NOTE**

> **This action is an in-place upgrade, and will REPLACE the current application. In case this feature does not work as intended or as the user may wish, it is advised to make a backup of your applications before using, or specify al alternate directory as you would when creating a new file.**

Reviewer: ```suggestion
**This action is an in-place upgrade, and will REPLACE the current application. In case this feature does not work as intended or as the user may wish, it is advised to make a backup of the app to be upgraded before using, or specify an alternate directory as you would when creating a new file.**
```
